chirp/chirp.go

```diff
@@ -10,6 +10,7 @@ import (
 	"bufio"
 	"fmt"
 	"net"
+	"regexp"
```

Comment:
FYI we try to avoid the regexp package for iOS/binary size reasons, but since this isn't on iOS, it's fine

Reply:
does darwin imply iOS?

``` 
// tailscaled_bird.go
//go:build linux || darwin || freebsd || openbsd
// +build linux darwin freebsd openbsd
```

Reply:
oh right, no tailscaled on iOS